back-end/admin-page/src/components/lab/labAddForm.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useRef } from "react";
+import { default as Button } from "../button/button";
+import { StyledForm, StyledLabel, StyledInput } from "./labAddForm.styled";
+
+const LabAddForm = ({ onAdd }) => {
+  const formRef = useRef();
+  const nameRef = useRef();
+  const onSubmit = (e) => {
+    e.preventDefault();
+    const lab = {
+      name: nameRef.current.value || "",
+    };
+    formRef.current.reset();
+    onAdd(lab);
+  };
+
+  return (
+    <StyledForm ref={formRef} onSubmit={(e) => e.preventDefault()}>
+      <StyledLabel>
+        NAME
+        <StyledInput ref={nameRef} type="text" name="name" />
+      </StyledLabel>
+      <Button text="등록하기" onSubmit={onSubmit} />
+    </StyledForm>
+  );
+};
```

Comment:
```suggestion
const LabAddForm = ({ onAdd }) => {
  const formRef = useRef();
  const nameRef = useRef();
  const onSubmit = (e) => {
    e.preventDefault();
    const lab = {
      name: nameRef.current.value || "",
    };
    formRef.current.reset();
    onAdd(lab);
  };

  return (
    <StyledForm ref={formRef} onSubmit={onSubmit}>
      <StyledLabel>
        NAME
        <StyledInput ref={nameRef} type="text" name="name" />
      </StyledLabel>
      <Button text="등록하기" />
    </StyledForm>
  );
};
```

onSubmit은 form에다가 달면 조아요~ submit Event는 Form Element의 event 입니다 :smiley_cat: 

https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event

Reply:
오호!! 그렇군요!  감사합니다 :)